README.md

```diff
@@ -748,7 +748,7 @@ put_rdfmap("path/to/file", "rdf-map", target: "<rdfProperty>")
 lookup("path.to.field", "rdf-map")
 
 # with default value
-lookup("path.to.field", "map-name", __default: "NA")
+lookup("path.to.field", "map-name", default: "NA")
```

Comment:
The option needs to be written in quotation marks because it is a reserved word in Java.

Reply:
```suggestion
lookup("path.to.field", "map-name", "default": "NA")
```

Reply:
Done. Thanks